luma/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017 Richard Hull and contributors
+# See LICENSE.rst for details.
+
+from pkgutil import extend_path
```

Comment:
can you add a comment to these lines, because it's not immediately clear to me why this is necessary.